app/controllers/api/subscriptions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::SubscriptionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    payment_status = perform_payment
```

Comment:
I know this is my naming but you should question that because it is not very readable

Reply:
I'll try to be better with naming. Thanks